### ISSUE TITLE
Bug 54060914: [React Native Gallery -> Clipboard]: Accessible name and visible name is defined incorrectly for ‘Copy text to the Clipboard’ button.

### ISSUE THREAD

**Issue by Yajur-Grover** [microsoft/react-native-gallery#493]
Build Details:
OS Version: Dev (OS Build 27695.1000)
App: React Native Gallery 
App Version: 1.0.18.0
React Native Windows Version: 0.75.0
AT: Screen Reader
 
Pre-Requisite:
Turn on Narrator (Ctrl + Win + Enter).
 
Repro-Steps:
Launch the React Native Gallery app.
Navigate to ‘All samples’ button present in left navigation pane and invoke it.
Navigate to ‘Clipboard’ button present under “System” group and invoke it.
Navigate to ‘Copy text to the Clipboard’ button and listen to narrator announcement.
Actual Result:
Accessible name and visible name is defined incorrectly for ‘Copy text to the Clipboard’ button, when user navigated to ‘Copy text to the Clipboard’ button, narrator is announcing as “Example copy text, button".
 
Note: Same issue is observed ‘Paste text from the Clipboard’ button.
 
Expected Result:
Accessible name and visible name should be defined properly for ‘Copy text to the Clipboard’ button and when user navigates to it, narrator should announce as “Copy text to the Clipboard, button".
 
User Impact:
Users with visual impairments who rely on screen reader will not be able to know the exact name of the button if accessible name and visible name are defined incorrectly, they may get confused if narrator is announcing different names.

Link to ADO bug: https://dev.azure.com/microsoft/OS/_workitems/edit/54060914
